R/adjust.R

```diff
@@ -6,29 +6,38 @@
 #' @param effect Character vector of column names to be adjusted for (regressed out). If \code{NULL} (the default), all variables will be selected.
 #' @inheritParams standardize
 #' @param multilevel If \code{TRUE}, the factors are included as random factors. Else, if \code{FALSE} (default), they are included as fixed effects in the simple regression model.
+#' @param additive If \code{TRUE}, continuous variables as included as smooth terms in additive models. The goal is to regress-out potential non-linear effects.
 #' @param bayesian If \code{TRUE}, the models are fitted under the Bayesian framework using \code{rstanarm}.
```

Comment:
@DominiqueMakowski What priors are used? Default (what are they)? Are we not true Bayesians??!

Reply:
rstanarm's defaults... (in my defence this function is very exploratory :) but yeah further down the road it would be useful to tweak priors, especially the capacity of increasing precision to get a "regularized" partialization

Reply:
(I speak of partialization because the main use case for this function for now will be partial correlations in correlation)